src/Microsoft.Azure.SignalR.Common/ServiceConnections/ServiceConnectionContainerBase.cs

```diff
@@ -598,14 +607,15 @@ public void Stop()
                     }
                     if (Interlocked.Decrement(ref _counter) == 0)
                     {
-                        _timer.Stop();
+                        _timer?.Stop();
```

Comment:
`_counter` already ensure `_timer = Init()`, no need to check null here?

Reply:
yeah, pretty much being too defensive..